workflows/release-helm-charts.yml

```diff
@@ -0,0 +1,109 @@
+---
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Intel Corporation
+# Copyright 2024 Kyunghee University
+# Copyright 2024 The Linux Foundation
+
+name: Publish image and tag/release code
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  version-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: check version
+        run: |
+          sudo snap install yq
+          export COMPARISON_BRANCH=origin/master
+          git branch -a
+          make check-version
```

Comment:
It looks like this target exists in the `sdran-helm-charts` but I do not see it in the `roc-helm-charts`, correct? Or am I overlooking something?

Reply:
Nope, you're correct. I've been working a lot on Voltha, which uses a standardized library of makefiles, and just totally forgot to double-check here. But it's a simple call to the version_check.sh script, so I just replaced it with a direct call.